service.libraryautoupdate/resources/language/Afrikaans/strings.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
and if you update follow this http://kodi.wiki/view/language_support#Layout_of_the_language_folders